.github/workflows/Dev-Promotion.yml

```diff
@@ -0,0 +1,100 @@
+---
+name: Dev promotion
+
+on: 
+#  push:
+#    branches: 
+#      - master
+#    paths-ignore:
+#      - '.github/workflows/**'
+#      - 'test-script'
+#      - '**/pom.xml' 
+  workflow_dispatch:
+
+env:
+  IMAGE_BASE_NAME: ceres-test
+  HELM_CHART: ceres
+  HELM_CHART_FOLDER: helm-ceres/cherts/ceres
+  CLUSTER_NAME: primary-cluster
+  REGISTRY: gcr.io
+
+jobs:
+  Build:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.setVersion.outputs.version }}
+    steps:
+    ### Seting up runner env ###
+      - uses: actions/checkout@v3
+
+      - uses: ./.github/actions/GCPSetup
+        with:
+          GCPCreds: ${{ secrets.GCP_SA_JSON }} 
+
+      - name: 'Maven settings'
+        run: 'gsutil cp gs://salus-mavenrepository/m2-settings.xml ./.mvn/settings.xml'
+
+      - name: Docker login
+        run: gcloud auth configure-docker ${{ env.REGISTRY }} -q
+
+      - name: .m2 forder [*** fix it please ***]
+        run: sed -i '/<localRepository>/d' ./.mvn/settings.xml
+
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          java-version: '14'
+          distribution: 'adopt-hotspot'
+          cache: maven
+
+      - name: Set version
+        id: setVersion
+        shell: bash
+        run: |
+          POM_VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
+          RUN_ID=${{ github.run_number }}
```

Comment:
Should this be `run_attempt`? The `run_number` doesn't change when you trigger a re-run of a workflow. Since this is just checking out master and building what's there, you risk incorrectly overwriting old images with newer code.

Reply:
Using run_attempt will set all build numbers to 1, as it resets every time a new action run is trigger, run_number give us more traceability options, not only to code but to the build run too.

Reply:
ah, so it should be `${{run_number}}.${{run_attempt}}` probably, to keep them unique.